packages/components/src/components/loading/loading.tsx

```diff
@@ -1,9 +1,15 @@
+import React, { HTMLProps } from 'react';
 import classNames from 'classnames';
-import React from 'react';
-import PropTypes from 'prop-types';
 import Text from '../text/text';
 
-const Loading = ({ className, id, is_fullscreen = true, is_slow_loading, status, theme }) => {
+type TLoadingProps = HTMLProps<HTMLDivElement> & {
```

Comment:
```suggestion
export type TLoadingProps = HTMLProps<HTMLDivElement> & {
```
@jim-deriv I suggest importing TLoadingProps in button_loading.tsx 🙂